### PR TITLE
#1: Add support for Github Enterprise instances

### LINKED ISSUE #1: Add Enterprise Suppport
Hello! This is a really great tool that I came across and could definitely use right now. I see that it currently only supports the standard github URL, I was wondering if I could request or work on adding in support for enterprise instances?

Thanks!

### PR DESCRIPTION
**Pull request by chrisschaaf**
Fixes #1 

I also added shorthand aliases for each of the click options.

### CODE REVIEW

**Comment by chrisschaaf** [comment]
Seems to be working as expected. Command I tested with:

```sh
./scripts/run --organization ETS-Alpha --find "<redacted>" --replace "<redacted>" -b github.cerner.com
```
Beginning of output: (not sharing full output since some of this is probably not useful and not able to be shared)
```
Found matches in 47 repos.
See potential changes? [y/N]: y
Repository(full_name="ETS-Alpha/ets_alpha_chef-repo")
...
```
Help now shows:
```
❯ ./scripts/run --help
Usage: main.py [OPTIONS]

Options:
  -b, --base-url TEXT             Github Enterprise URL, example:
                                  'github.company.com'
  -o, --organization TEXT         [required]
  -f, --find TEXT                 [required]
  -r, --replace TEXT              [required]
  -e, --extra-search-params TEXT
  --help                          Show this message and exit.
```

File: main.py [hunk 1]
```diff
@@ -68,13 +68,15 @@ def create_pr(self, message, branch_name, labels):
 
 
 @click.command()
-@click.option("--organization", type=str, required=True)
-@click.option("--find", type=str, required=True)
-@click.option("--replace", type=str, required=True)
-@click.option("--extra-search-params", type=str, default="")
+@click.option('-b', '--base-url', type=str, default="", help='Github Enterprise URL, example: \'github.company.com\'')
```
Comment (davegaeddert) [inline main.py:71]: I wonder if "hostname" would make more sense here since you're constructing the full url later? Maybe `--ghe-hostname`?

Comment (chrisschaaf) [reply main.py]: That would probably be less confusing than sharing the `base_url` attribute name from the Github component, good idea!

**Comment by davegaeddert** [comment]
Awesome. This looks totally reasonable to me -- I'll merge it in. Thanks for beefing up this little script!

**Comment by chrisschaaf** [comment]
Thanks @davegaeddert!

**Pull request merged**